docs/current/faq.md

```diff
@@ -151,6 +151,15 @@ Follow these steps:
     - Python: [https://github.com/platformdirs/platformdirs](https://github.com/platformdirs/platformdirs)
   :::
 
+### What are Dagger's caching features?
```

Comment:
This is a first cut, would like inputs from @gerhard @vito @jpadams @marcosnils @shykes @samalba on this one.

Reply:
Looks good to me @vikram-dagger 👍

Will merge & publish with a bunch of other docs changes that are still in flight.